docs/production-report/README.md

```diff
@@ -1,374 +1,323 @@
+Change the **Coloring** from **Status color** to **Late in Red**. This will show tasks in **Grey** if they are on time and **Red** if they are late.
 
 ![Task Type Schedule Late](../img/getting-started/task_type_schedule_coloring_late.png)
 
+You can return to the **Tasks** tab for more details, and Kitsu will retain your filters from tab to tab.
 
-You can return to the **Tasks** tab for more details. Kitsu will keep you filtered from tab to tab.
 
-  
-## How to Know if a Task Type may be late
- 
-Realizing that a task is late is good, but knowing if it may be late is better. As Production, you always need to be ahead.
+## Understanding Why a Task is Late
 
-On the **Task Type** page, use the **Due Date Status** filter.
+Now that you know which tasks are late or will be late, you need to understand **why**. There are several reasons why a task might be late:
 
-Depending on how much you want to know, you can focus on the **Due this week** and the **Due this month** filters.
+- The artist is overwhelmed with too many tasks.
+- There is too much back-and-forth on the task.
+- The task might be underestimated, making it difficult to finish on time.
+- The previous task was already late.
 
+### Checking an Artist's Workload
 
-![Task Type Due this Month](../img/getting-started/task_type_due_month.png)
-
-
-You now have a global view of what will happen next. For a more precise view, feel free to use the Status filter.
-
-For example, if you are on the **Due this week** view, you may want to see if there are still **Todo** or **Ready** tasks. 
-There will likely be late if a task needs to be finished this week and has yet to start. 
-
-
-
-
-
-To be sure, you must also check the amount of work assigned to the artists who still need to start.
-Go to the **Estimation** tab with your filters still active.
-
-The sum-up of your artists is on the **Estimation** tab on the right part of the screen.
-
-![Task Type Due this Week Estimation](../img/getting-started/task_type_due_week_estimation.png)
-
-
-The first line shows the total number of tasks assigned to the Artist since the beginning of the Production. The second line, **Remaining, ** shows what the Artist has left to do. 
-
-The artist may miss the deadline if he still has plenty of tasks. If he still needs to start to do it, there is still a chance. But you need to keep an eye on those tasks.
-
-
-## How to Know Why a Task is Late
-
-Now that you know which tasks are late or will be late, you need to understand **why**.
-
-There are several reasons why a task is late: 
-- The Artist is overwhelmed and has too many tasks to do at the same time
-- There is too much back-and-forth on this Task
-- The task must be underestimated and be easier to finish on time.
-- The previous Task was already late
-
-To check an artist's number of tasks, you can filter the **Task Type** page by the artist's name under the **Task** tab.
```

Comment:
I think we should keep this contextualizing paragraph.

Reply:
This was re-written into a new section which is found at line 159 - 164

It has been changed slightly so let me know if it still doesn't make sense